conda/e3sm_diags_env.yml

```diff
@@ -11,6 +11,7 @@ dependencies:
   - matplotlib=3.4.2
   - beautifulsoup4=4.9.3
   - cartopy=0.19.0
+  - cartopy_offlinedata
```

Comment:
@tomvothecoder Do we need to pin a specific version? It looks like v0.2.4 is the newest release on https://github.com/willirath/cartopy_offlinedata/releases

Reply:
I would not pin a specific release.  The data doesn't typically change. The latest release was a small bug fix to how logging is handled when it warns you that your data is coming from this package rather than being downloaded.